keyboards/kprepublic/cstc40/daughterboard/daughterboard.c

```diff
@@ -0,0 +1,115 @@
+    { 0, L_6,  K_6,  J_6 },
+	{ 0, L_7,  K_7,  J_7 },
+    { 0, L_8,  K_8,  J_8 },
+    { 0, L_9,  K_9,  J_9 },
+    { 0, L_10, K_10, J_10 },
+    { 0, L_11, K_11, J_11 },
+    { 0, L_12, K_12, J_12 },
+
+    { 0, I_1,  H_1,  G_1 },
+    { 0, I_2,  H_2,  G_2 },
+    { 0, I_3,  H_3,  G_3 },
+    { 0, I_4,  H_4,  G_4 },
+    { 0, I_5,  H_5,  G_5 },
+    { 0, I_6,  H_6,  G_6 },
+    { 0, I_7,  H_7,  G_7 },
+    { 0, I_8,  H_8,  G_8 },
+    { 0, I_9,  H_9,  G_9 },
+    { 0, I_10, H_10, G_10 },
+    { 0, I_11, H_11, G_11 },
+    { 0, I_12, H_12, G_12 },
+
+    { 0, F_1,  E_1,  D_1 },
+    { 0, F_2,  E_2,  D_2 },
+    { 0, F_3,  E_3,  D_3 },
+    { 0, F_4,  E_4,  D_4 },
+    { 0, F_5,  E_5,  D_5 },
+    { 0, F_6,  E_6,  D_6 },
+    { 0, F_7,  E_7,  D_7 },
+    { 0, F_8,  E_8,  D_8 },
+    { 0, F_9,  E_9,  D_9 },
+    { 0, F_10, E_10, D_10 },
+    { 0, F_11, E_11, D_11 },
+    { 0, F_12, E_12, D_12 },
+
+    { 0, C_1, B_1, A_1 },
+    { 0, C_2, B_2, A_2 },
+    { 0, C_3, B_3, A_3 },
+    { 0, C_4, B_4, A_4 },
+    { 0, C_5, B_5, A_5 },
+    { 0, C_6, B_6, A_6 },
+    { 0, C_8, B_8, A_8 },
+    { 0, C_9, B_9, A_9 },
+    { 0, C_10, B_10, A_10 },
+    { 0, C_11, B_11, A_11 },
+    { 0, C_12, B_12, A_12 }
+};
+
+led_config_t g_led_config = {
+    {
+        // Key Matrix to LED Index
+        { 0,  1,  2,  3,  4,  5,      6,  7,  8,  9, 10, 11},
+        {12, 13, 14, 15, 16, 17,     18, 19, 20, 21, 22, 23},
+        {24, 25, 26, 27, 28, 29,     30, 31, 32, 33, 34, 35},
+        {36, 37, 38, 39, 40, 41, NO_LED, 42, 43, 44, 45, 46}
+    }, {
+        // LED Index to Physical Position
+        {  0,  0}, { 20,  0}, { 40,  0}, {61,  0}, {81,  0}, {101,  0}, {122,  0}, {142,  0}, {162,  0}, {183,  0}, {203,  0}, {224,  0},
+        {  0, 21}, { 20, 21}, { 40, 21}, {61, 21}, {81, 21}, {101, 21}, {122, 21}, {142, 21}, {162, 21}, {183, 21}, {203, 21}, {224, 21},
+        {  0, 42}, { 20, 42}, { 40, 42}, {61, 42}, {81, 42}, {101, 42}, {122, 42}, {142, 42}, {162, 42}, {183, 42}, {203, 42}, {224, 42},
+        {  0, 64}, { 20, 64}, { 40, 64}, {61, 64}, {81, 64},       {111, 64},      {142, 64}, {162, 64}, {183, 64}, {203, 64}, {224, 64},
+    }, {
+        // LED Index to Flag
+        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+        1, 1, 1, 1, 1,   1,  1, 1, 1, 1, 1
+    }
+};
+#endif
+
+// Alpha flagged layer indicator
+//bool rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
+//    for (uint8_t i = led_min; i < led_max; i++) {
+//		if (g_led_config.flags[i] & LED_FLAG_MODIFIER) {
+//			switch(get_highest_layer(layer_state|default_layer_state)) {
+//				case 1:
+//					rgb_matrix_set_color(i, RGB_ORANGE);
+//					break;
+//				case 2:
+//					rgb_matrix_set_color(i, RGB_PURPLE);
+//					break;
+//				case 3:
+//					rgb_matrix_set_color(i, RGB_RED);
+//					break;
+//				default:
+//					break;
+//			}
+//        }
+//    }
+//	return false;
+//}
```

Comment:
```suggestion
#endif
```
This can be moved to `info.json` and the commented-out code shouldn't be here at the keyboard level anyway.